study/Cloud Computing/k8s-files/keycloak/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Use the official Keycloak image as the base
+FROM quay.io/keycloak/keycloak:latest as builder
+
+# Enable health and metrics
+ENV KC_HEALTH_ENABLED=true
+ENV KC_METRICS_ENABLED=true
+
+# Configure database vendor
+ENV KC_DB=postgres
+
+WORKDIR /opt/keycloak
+
+# Generate a demo key store (not for production)
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 \
+    -dname "CN=server" -alias server \
+    -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
```

Comment:
IP:127.0.0.1
Is this ok?
I still dont trus the certificate created that way, we might also need a conf file that says that the host machine (my windows in this case) is also allowed to forward traffic

Reply:
removed all this as the certificate is created outside